src/pages/lessons/[slug].tsx

```diff
@@ -116,44 +116,41 @@ type LessonProps = {
 }
 
 const MAX_FREE_VIEWS = 7
+const HEADER_HEIGHT = 80
 
 const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
+  const router = useRouter()
+
+  const {viewer} = useViewer()
+  const {setPlayerPrefs, playbackRate, defaultView} = useEggheadPlayerPrefs()
+
   const {md} = useBreakpoint()
-  const {
-    setPlayerPrefs,
-    playbackRate: storedPlaybackRate,
-    defaultView,
-  } = useEggheadPlayerPrefs()
+
   const {height} = useWindowSize()
-  const [ref, {width: videoWidth}] = useMeasure<any>()
-  const HEADER_HEIGHT = 80
   const CONTENT_OFFSET = height < 450 ? 30 : 120
   const HEIGHT_OFFSET = HEADER_HEIGHT + CONTENT_OFFSET
+
   const [lessonMaxWidth, setLessonMaxWidth] = React.useState(0)
+  const [ref, {width: videoWidth}] = useMeasure<any>()
+
   const [media, setMedia] = React.useState<any>()
   const [isFullscreen, setIsFullscreen] = React.useState(false)
 
-  const [playbackRate, setPlaybackRate] = React.useState<number>(
-    storedPlaybackRate,
-  )
-  const changePlaybackRate = (rate: number) => {
-    setPlayerPrefs({playbackRate: rate})
-    setPlaybackRate(rate)
-  }
-
-  const [lesson, setLesson] = React.useState<any>(initialLesson)
-  const router = useRouter()
   const playerRef = React.useRef(null)
-  const {viewer} = useViewer()
-  const [playerState, send] = useMachine(playerMachine)
+  const actualPlayerRef = React.useRef<any>(null)
+
+  const [playerState, send] = useMachine(playerMachine, {
```

Comment:
with `useMachine` the second argument is `options` which let's us send in an initial context. In this case, we can use the initial lesson.